src/full-screen/components/runtime.tsx

```diff
@@ -0,0 +1,88 @@
+import React, { useCallback, useEffect } from "react";
+import _screenfull from "screenfull";
+import queryString from "query-string";
+import { FullScreenButton } from "./full-screen-button";
+import { useForceUpdate } from "../../shared/hooks/use-force-update";
+
+import css from "./runtime.scss";
+
+const screenfull = _screenfull.isEnabled ? _screenfull : undefined;
+
+// eslint-disable-next-line no-empty-pattern
+export const Runtime: React.FC = ({}) => {
```

Comment:
```suggestion
export const Runtime: React.FC = () => {
```
then you can eliminate `// eslint-disable-next-line no-empty-pattern` on the previous line.